app/services/katello/pulp/pulp_content_unit.rb

```diff
@@ -0,0 +1,55 @@
+require 'set'
+
+module Katello
+  module Pulp
+    class PulpContentUnit
+      #  Any class that includes this module should define:
```

Comment:
seemed kinda self-explanatory to me, but i guess previously there was more than one non-require method/constant.  add more verbose text